scene/gui/color_picker.cpp

```diff
@@ -992,6 +1034,9 @@ void ColorPickerButton::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_popup"), &ColorPickerButton::get_popup);
 	ClassDB::bind_method(D_METHOD("set_edit_alpha", "show"), &ColorPickerButton::set_edit_alpha);
 	ClassDB::bind_method(D_METHOD("is_editing_alpha"), &ColorPickerButton::is_editing_alpha);
+	ClassDB::bind_method(D_METHOD("_about_to_popup"), &ColorPickerButton::_about_to_popup);
+	ClassDB::bind_method(D_METHOD("_color_changed"), &ColorPickerButton::_color_changed);
+	ClassDB::bind_method(D_METHOD("_modal_closed"), &ColorPickerButton::_modal_closed);
```

Comment:
Why do those need to be bound again? Sounds like a forward porting issue from developing it against `3.2`?

Reply:
Fixed.